.github/workflows/ci.yml

```diff
@@ -1045,6 +1045,7 @@ jobs:
           !contains(github.event.pull_request.labels.*.name, 'tests:all-product')
         run: echo "PTL_OPTS=--impacted-features impacted-features.log" >> $GITHUB_ENV
       - name: Product Tests
+        id: tests
```

Comment:
product-tests ?

Reply:
It's not important to distinguish the kind of tests here, and it's better to keep it consistent across all jobs. Very few jobs have more than one step that runs tests, and it has a suffix then.